packages/api/src/useSubscription.ts

```diff
@@ -8,12 +8,14 @@ import type {
     TSocketSubscribableEndpointNames,
 } from '../types';
 
-const useSubscription = <T extends TSocketSubscribableEndpointNames>(name: T) => {
+const useSubscription = <T extends TSocketSubscribableEndpointNames>(name: T, idle_time = 5000) => {
     const [isLoading, setIsLoading] = useState(false);
     const [isSubscribed, setSubscribed] = useState(false);
+    const [isIdle, setIdle] = useState(false);
```

Comment:
```suggestion
    const [is_idle, setIdle] = useState(false);
```

Reply:
nit

Reply:
I wanted to follow the similar syntax used in ReactQuery here 👀

Reply:
oh I didn't check the rest of them
it's ok then 👍